Formula/clang-format@8.rb

```diff
@@ -0,0 +1,60 @@
+# keep older version due to incompatibility
+class ClangFormatAT8 < Formula
+  desc "Formatting tools for C, C++, Obj-C, Java, JavaScript, TypeScript"
+  homepage "https://clang.llvm.org/docs/ClangFormat.html"
+  url "https://github.com/llvm/llvm-project/releases/download/llvmorg-8.0.1/llvm-8.0.1.src.tar.xz"
+  sha256 "44787a6d02f7140f145e2250d56c9f849334e11f9ae379827510ed72f12b75e7"
+  version_scheme 1
```

Comment:
Remove this line; new formulae don't need `version_scheme`. Please also add an appropriate `license`.